cmd/server/main.go

```diff
@@ -7,18 +7,44 @@ import (
 	"net/http"
 	"path/filepath"
 	"time"
+
+	"github.com/tony/mot-server/cmd/server/api"
 	"github.com/tony/mot-server/cmd/server/middleware"
+	"github.com/tony/mot-server/cmd/server/model"
 	"github.com/tony/mot-server/cmd/server/router"
 
 	"github.com/gin-gonic/gin"
 	amqp "github.com/rabbitmq/amqp091-go"
 )
 
+var err error
+
+// init will run first then main
+// first we ensure all clients init with no errors
+func init() {
```

Comment:
main.go is first file. init() in main.go is the first func.
After we run main.go, here will be the first place. Then it's easy for us to know where is wrong if it panic